src/coreComponents/common/format/table/TableFormatter.cpp

```diff
@@ -169,7 +169,6 @@ string TableTextFormatter::toString( TableData const & tableData ) const
 
   outputLayout( tableOutput, columns, msgTableError, sectionSeparatingLine );
   outputSectionRows( columns, sectionSeparatingLine, tableOutput, nbValuesRows, TableLayout::Section::values );
-  tableOutput << '\n';
```

Comment:
won't this affect other tables as well?

Reply:
it will and I hate that additional line break

Reply:
okay, maybe a separate small PR for this would be better. Unless it's a known issue. What tables are these we are writing to the log?

Reply:
ok let me revert